Cargo.toml

```diff
@@ -23,6 +23,7 @@ alloc = ["rand_core/alloc"]  # enables Vec and Box support without std
 i128_support = [] # enables i128 and u128 support
 
 serde-1 = ["serde", "serde_derive"]
+wasm-stdweb = ["stdweb"]
```

Comment:
Shouldn't we just call it `stdweb`, or rather just use `cfg(feature="stdweb")` since that is already an optional dependency?

Reply:
That would be my choice, but cargo does not support features that have the same name as dependencies. We have the same problem with `serde`, which has `serde-1` as feature flag.

Reply:
I didn't know using the name of an optional dependency was possible, much better!